dist/debian/control.template

```diff
@@ -39,6 +39,12 @@ Description: Scylla kernel tuning configuration
  Scylla is a highly scalable, eventually consistent, distributed,
  partitioned row DB.
 
+Package: %{product}-node-exporter
+Architecture: any
+Conflicts: prometheus-node-exporter
```

Comment:
If node_exporter is available in debian, maybe we shouldn't package it there. I checked and it's available on Debian 10 and Ubuntu 16.04.

Reply:
Not sure about it. It can cause differences between debian and red hat,  but node_exporter is not something that changes all the time.

@penberg opinion?

Reply:
I think it's okay. We install it for .tar and .rpm. so we should also install it for .deb. It's optional, people who want the system package can install the system package.